### DISCUSSION TITLE
Excluded Apps is being overwritten at each Update

### DISCUSSION THREAD

**Discussion by MBstatsara** [Romanitho/Winget-Install#114] [General]
Hello
It seems as the wingetauto updater app patches to the latest version, the excluded apps file is being overwritten.
This is causing the excluded apps we have centrally to be older than the file it is currently using, so it doesnt use it.
This means we are having to update our excluded apps file with no changes to make it always being used.

How do we stop the excluded apps file being overwritten when a patch occurs?

**Comment by KnifMelti** [comment]
The issue was not present in today's pre-release version 1.20.2-1 for me, nor has it occurred for us previously with stable version releases.
Within the `Update-WAU.ps1` script, the `WAU.zip` that is downloaded includes the file `excluded_apps.txt`. This file is situated outside the `$WorkingDir\WAU_update\Winget-Autoupdate` folder, which is the directory from where the new files are sourced.

**Reply by MBstatsara** [reply]
Sorry have moved this to the autoupdate rep, as didnt realise i was in the installer section.
Was fine on all other builds but can confirm this latest patch last night 
WAU 1.20.2-1 is the build we moved to..

Previously the excluded apps file hasnt been overwritten, but i have about 50 PCs where it did today, i had to edit the central repo again to get the file changed back

**Reply by KnifMelti** [reply]
You shouldn't use pre-release for production...
...and I didn't have this issue with my central repo (https).
@Romanitho ?

**Reply by MBstatsara** [reply]
There is no option to disable pre-release?

**Reply by KnifMelti** [reply]
See https://github.com/Romanitho/Winget-AutoUpdate/discussions/650#discussioncomment-10084361
